lib/shared/addon/components/cluster-driver/driver-linodelke/template.hbs

```diff
@@ -0,0 +1,341 @@
+            </label>
+            {{#input-or-display editable=true value=selectedNodePoolType}}
+              {{new-select
+                class="form-control"
+                content=nodePoolChoises
+                value=selectedNodePoolType
+              }}
+            {{/input-or-display}}
+          </div>
+          <div class="col span-1">
+            <div class="acc-label pb-15">
+              {{t "clusterNew.linodelke.nodePoolConfig.monthly"}}:
+            </div>
+            {{format-number this.selectedNodePoolObj.price.monthly style='currency' currency='USD' allowEmpty=true}}
+          </div>
+          <div class="col span-1">
+            <div class="acc-label pb-15">
+              {{t "clusterNew.linodelke.nodePoolConfig.hourly"}}:
+            </div>
+            {{format-number this.selectedNodePoolObj.price.hourly style='currency' currency='USD' allowEmpty=true}}
+          </div>
+          <div class="col span-1">
+            <div class="acc-label pb-15">
+              {{t "clusterNew.linodelke.nodePoolConfig.ram"}}:
+            </div>
+            {{this.selectedNodePoolObj.memoryGb}}
+            {{t "generic.gigabyte"}}
+          </div>
+          <div class="col span-1">
+            <div class="acc-label pb-15">
+              {{t "clusterNew.linodelke.nodePoolConfig.cpus"}}:
+            </div>
+            {{this.selectedNodePoolObj.vcpus}}
+          </div>
+          <div class="col span-1">
+            <div class="acc-label pb-15">
+              {{t "clusterNew.linodelke.nodePoolConfig.storage"}}:
+            </div>
+            {{this.selectedNodePoolObj.diskGb}}
+            {{t "generic.gigabyte"}}
+          </div>
+          <div class="col span-1">
+            <div class="acc-label">
+              {{t "clusterNew.linodelke.nodePoolConfig.count"}}:
+            </div>
+            <Input
+              @type="number"
+              @min="1"
+              @value={{this.selectedNodePoolObj.count}}
+            />
+          </div>
+          <div class="col span-1">
+            <div class="acc-label pb-10">
+              {{t "clusterNew.linodelke.nodePoolConfig.actions"}}
+            </div>
+            <button
+              class="btn bg-primary icon-btn p-0"
+              {{action "addSelectedNodePool"}}
+            >
+              <span class="darken">
+                <i class="icon icon-plus text-small"></i>
+              </span>
+              <span>
+                {{t "clusterNew.linodelke.nodePoolConfig.addNodePool"}}
+              </span>
+            </button>
+          </div>
+        </div>
+        <div class="ember-view">
+          <div class="header mt-20 clearfix">
+            <div class="pull-left">
+              <h2 class="mb-0">
+                {{t "clusterNew.linodelke.nodePools.label"}}
+              </h2>
+            </div>
+          </div>
+          <div class="grid sortable-table ember-view">
+            <table class="fixed grid sortable-table">
```

Comment:
I'm apologize, I seem to have missed this on the first review. I am not sure if you intentionally excluded it? If not, we have a component called `sortable-table` and this seems like a great use case. `sortable-table` is used extensively throughout the app but if you have any questions about implementation, feel free to reach out.